src/eventhub.ts

```diff
@@ -434,30 +429,30 @@ class Eventhub extends EventEmitter implements IEventhub {
    * List all current subscriptions.
    * @return Array containing current subscribed topics.
    */
-  public listSubscriptions() : Promise<any> {
+  public listSubscriptions(): Promise<any> {
     return this._sendRPCRequest(RPCMethods.LIST, []);
   }
 
   /**
    * Get key from key/value store.
    * @param key Key to fetch.
    */
-  public get(key: string) : Promise<any> {
+  public get(key: string): Promise<any> {
     return this._sendRPCRequest(RPCMethods.GET, { key: key });
   }
 
   /**
    * Set key in key/value store.
    * @param key Key to set.
    */
-  public set(key: string, value: string, ttl?: number) : Promise<any> {
-    let setRequest = {
-      key: key,
-      value: value
-    }
+  public set(key: string, value: string, ttl?: number): Promise<any> {
+    const setRequest: Record<string, any> = {
```

Comment:
Eventhub expects { key: "keyName", value: "valueHere" }. This will only send { keyName, valueHere } from my understanding. Or am I missing something here ? :)